drawBot/context/imageObjectContext.py

```diff
@@ -0,0 +1,43 @@
+import tempfile
+
+from PIL import Image
+import io
+import AppKit
+
+from .imageContext import ImageContext
+
+class BaseImageObjectContext(ImageContext):
+
+    def _writeDataToFile(self, data, path, options):
+        self._imageObjects = []
+        with tempfile.NamedTemporaryFile(suffix=".tiff") as temp:
```

Comment:
Is there any way we can do thins without a temp file?

Reply:
possible, now nothing is written to disk, let me check